addon/styles/_frost-expand.scss

```diff
@@ -0,0 +1,56 @@
+$frost-color-expand-label-text: $frost-color-blue-1 !global;
+$frost-color-expand-scroll-border: $frost-color-lgrey-3 !global;
+$frost-color-expand-content-bg: $frost-color-lgrey-4 !global;
+$frost-color-expand-content-text: $frost-color-grey-3 !global;
+
+.frost-expand {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-family: $frost-font-family;
+
+  &.collapsed {
+    .frost-expand-label-icon {
+      transform: rotate(-90deg);
+    }
+  }
+
+  &.expanded {
+    .frost-expand-label-icon {
+      transform: rotate(0deg);
+    }
+  }
+
+  &-label {
+    display: flex;
+    flex-direction: row;
+    flex-shrink: 0;
+    justify-content: flex-start;
+    margin-bottom: 5px;
+    cursor: pointer;
+
+    &-text {
+      padding-left: 10px;
+      color: $frost-color-expand-label-text;
+    }
+
+    &-icon {
+      width: 20px;
+      height: 20px;
+      transition: transform .2s;
+    }
+  }
+
+  &-scroll {
+    border: 1px ridge $frost-color-expand-scroll-border;
```

Comment:
Can you validate `ridge` with the visual designers?  I don't think it's going to get sign-off, but I could be wrong